activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/Application.java

```diff
@@ -0,0 +1,21 @@
+package org.activiti.spring.boot;
+
+import org.activiti.runtime.api.connector.Connector;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class);
+    }
+
+
+    //@TODO: add tests for
```

Comment:
Is that still pending? Can we remove these comments?

Reply:
@erdemedeiros Let's remove the comments but let's create some issues to create this tests